docs/reference/configuration.md

```diff
@@ -394,6 +394,7 @@ Update strategy which will wait until a maximum of ``batch_size`` number of inst
 | object                       | type     | description
 | ---------------------------- | :------: | ------------
 | ```batch_size```             | Integer  | Maximum number of shards to be updated in one iteration (Default: 1)
+| ```autopause_after_batch```             | Boolean  | Automatically pauses update before starting a new batch. (Default: False)
```

Comment:
Given the flag name refers to what to do "after batch", the update pauses after the last batch, but not before the first one; I think it would be more clear to rephrase the description to something along "pauses the update after completing a batch"

Reply:
That reads a lot better, thanks! I was struggling to find a good way to describe the mechanism at the time but couldn't land on something good. Your description nails it.